src/main/groovy/edu/oregonstate/mist/locations/frontend/LocationConfiguration.groovy

```diff
@@ -0,0 +1,27 @@
+package edu.oregonstate.mist.locations.frontend
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import edu.oregonstate.mist.api.Credentials
+import io.dropwizard.Configuration
+import io.dropwizard.client.JerseyClientConfiguration
+
+import javax.validation.Valid
+import javax.validation.constraints.NotNull
+
+class LocationConfiguration extends Configuration {
+    @JsonProperty('authentication')
+    @NotNull
+    @Valid
+    List<Credentials> credentialsList
```

Comment:
instead of copying and pasting these lines from `edu.oregonstate.mist.api.Configuration`, i suggest importing that class and inheriting them. compare to https://github.com/osu-mist/directory-api/blob/feature/co-215-get/src/main/groovy/edu/oregonstate/mist/directoryapi/DirectoryApplicationConfiguration.groovy#L3